pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/LLCRealtimeClusterSplitCommitIntegrationTest.java

```diff
@@ -41,4 +48,57 @@ public void startServer() {
     serverConfig.setProperty(CommonConstants.Server.CONFIG_OF_ENABLE_SPLIT_COMMIT, true);
     startServer(serverConfig);
   }
+  // The goals of the following tests to show the diff between org.apache.commons.httpclient.URI and java.net.URI.
```

Comment:
Thanks so much for demonstrating this in an automated fashion. Yes, you can remove these tests in your next iteration.